modules/services/vdirsyncer.nix

```
@@ -67,9 +67,10 @@ in {
       Service = {
         Type = "oneshot";
         # TODO `vdirsyncer discover`
-        ExecStart = "${cfg.package}/bin/vdirsyncer ${
-            concatStringsSep " " vdirsyncerOptions
-          } sync";
+        ExecStart = [
+          "${cfg.package}/bin/vdirsyncer ${concatStringsSep " " vdirsyncerOptions} metasync"
+          "${cfg.package}/bin/vdirsyncer ${concatStringsSep " " vdirsyncerOptions} sync"
+        ];
```

I think the formatter is failing due to long lines. Maybe something like this?

```suggestion
        ExecStart = let
          optStr = concatStringsSep " " vdirsyncerOptions;
        in [
          "${cfg.package}/bin/vdirsyncer ${optStr} metasync"
          "${cfg.package}/bin/vdirsyncer ${optStr} sync"
        ];
```

---

Yeah, I forgot to run the formatter. Used your suggestion, though not with the exact same formatting since `./format` preferred
```nix
let optStr ...
in [ ...
```
instead.